vita3k/gui/src/settings_dialog.cpp

```diff
@@ -447,22 +448,20 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
     ImGui::Begin("##settings", &settings_dialog, ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_AlwaysAutoResize | ImGuiWindowFlags_NoSavedSettings);
     ImGui::PushFont(gui.vita_font);
     ImGui::SetWindowFontScale(0.7f * RES_SCALE.x);
-    auto settings_str = lang["title"].c_str();
+    auto settings_str = lang.main_window["title"].c_str();
     const auto title = is_custom_config ? fmt::format("{}: {} [{}]", settings_str, get_app_index(gui, emuenv.app_path)->title, emuenv.app_path) : settings_str;
     ImGui::SetCursorPosX((ImGui::GetWindowSize().x / 2.f) - (ImGui::CalcTextSize(title.c_str()).x / 2.f));
     ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%s", title.c_str());
-    ImGui::PopFont();
     ImGui::Spacing();
     ImGui::Separator();
-    ImGui::SetWindowFontScale(1.f * RES_SCALE.x);
```

Comment:
Why do you remove the current font? It honestly look a lot better with the vita font (at least from my point of view).
Also, this is not related to the topic of this PR.

Reply:
if not touch this, here looks like text is too big